backend/extra/intro-to-express/starter/src/routes/posts.js

```diff
@@ -1,7 +1,18 @@
 import express from "express";
 import logger from "../middleware/logger.js";
 import { getPosts } from "../controllers/postsController.js";
+import { postValidator, putValidator } from "../middleware/validation.js";
+import { creatPost, deletPost, getPostById, getPosts, updatPost } from "../controllers/postsController.js";7
+
 
 export const routes = express.Router();
 
+
+routes.get("/:id", getPostById);
 routes.get("/", logger, getPosts);
+routes.get("/", getPosts);
```

Comment:
@yasminebounasla hna kayn small repetition, you can remove the 1st one 3adi
```suggestion
routes.get("/", getPosts);
```